sparkler-api/src/main/java/edu/usc/irds/sparkler/model/FetchedData.java

```diff
@@ -17,75 +17,85 @@
 
 package edu.usc.irds.sparkler.model;
 
-
-import org.apache.hadoop.conf.Configuration;
-import org.apache.nutch.metadata.Metadata;
-
 import java.io.Serializable;
 import java.util.Date;
 
 public class FetchedData implements Serializable {
 
-    private Resource resource;
-
-	private byte[] content;
-	private String contentType;
-    private Integer contentLength;
-    private String[] headers;
-    private Date fetchedAt;
-    private Metadata metadata;
-	private int responseCode;
-
-
-	public FetchedData() {
-	}
-
-	public FetchedData(byte[] content, String contentType, int responseCode) {
-		super();
-		this.content = content;
-        this.contentLength = content.length;
-		this.contentType = contentType;
-		this.responseCode = responseCode;
-        this.metadata = new Metadata();
-	}
-	
-	public String getContentType() {
-		return contentType;
-	}
-	
-	public int getResponseCode() {
-		return responseCode;
-	}
-
-    public Resource getResource() { return resource; }
-
-    public byte[] getContent() {
-        return content;
+    private Resource _resource;
+    private byte[] _content;
+    private String _contentType;
+    private Integer _contentLength;
+    private String[] _headers;
+    private Date _fetchedAt;
+    private int _responseCode;
+    private String[] _outlinks;
+    private String _html;
+    private int _lastOutlinkInserted;
```

Comment:
When I used "scala friendly", I was not talking about coding conventions. And indeed, you can strongly disagree with this name convention which is not influenced by any open source style of coding convention. Could you provide me some references ?